roles/chaos-beaver/files/kill-container.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# kill-container.sh - ungracefully stops a docker container
+# Usage: kill-container.sh CONTAINER_NAME
+
+set -eu
+
+container_name=$1
+
+docker kill $container_name
```

Comment:
How were you planning to have jenkins pick which container to kill? Should this script instead pick a random container to kill, instead of relying on an argument to be passed?